pages/samvera/developer_resources/collections/hyrax_2.1/participants.md

```diff
@@ -0,0 +1,203 @@
+- [ ] Works are assigned participant access where managers are given edit_access and viewers are given read_access to any work created directly in the collection.
+
+
+These additional settings allow for backward compatibility for pre-defined collection types.
+
+* User Collection type shares permissions for the collection only, but does not apply permissions to works.
+* Admin Set type does not share view permissions to the admin set, but does apply permissions to works.
+* The default is for all to be checked.
+
+
+----
+
+## Understanding Collection Participants
+
+### How to set participants for a collection?
+
+* Dashboard -> Collections
+* Click Edit collection in action menu beside the collection you want to update
+* Select Sharing tab
+* Search for user/group
+* Set access level to Manager or Depositor or Viewer
+* Click Add button
+
+
+### Manager
+
+Managers of a collection can add to and remove works from the collection, modify collection metadata, and delete the collection.  If configured to set work permissions, the managers are given 'edit_access' to any work created directly in the collection.
+
+**Default:**  user creating the collection, Collection Type managers for this collection's type
+
+**Implementation:**
+
+* When a manager is added
+  * an entry is added to `permission_template_accesses` database table granting the user/group :manage access 
+  * `edit_access_group_ssim` or `edit_access_person_ssim` is set in the solr document
+  * a `Hydra::AccessControls::Permission` is added in Fedora
+* When a manager is removed
+  * the entry in `permission_template_accesses` database table is removed for this manager 
+  * `edit_access_group_ssim` or `edit_access_person_ssim` is updated to remove the group/user id from the solr document
+  * the `Hydra::AccessControls::Permission` is removed from Fedora
+* When the create new work actor stack is run, if this is the only collection assigned, then the work is considered to be created directly in the collection.  When a work is created directly in a collection 
+  * each manager is granted `edit_access` to the new work
+  
+NOTE: Access to a work is granted when the work is created.  Changes to managers after a work is created do not change the access grants of any existing works.  Work access grants can be updated in the work edit form to remove/add access for any group/user including those set by the collection.
+
+Special Note on Admin Sets:
+* A site can set managers on an admin set, which grants them :manage access to that admin set.  This creates the solr edit access and Fedora AccessControl entries as well, same as for other collections.
+* All works created with this admin set assigned at create time give the admin set's managers `edit_access` to the new work.
+
+
+
+### Depositor
+
+Depositors of a collection can view the collection's admin show page and add/remove works to it, even if the visibility permissions of the collection otherwise would not permit them to view it.
```

Comment:
I think the terminology here could use some tweaking.  "Depositor" does sound like someone who could add works. But "removing" works isn't something I'd expect a depositor to do.

> even if the visibility permissions of the collection otherwise would not permit them to view it

Can we be specific here about what "it" refers to? Is that the collection or the work?

Reply:
Good catch.  I removed "removing" form the description.  The updated description is in collection-sharing.md